filebeat/docs/fields.asciidoc

```diff
@@ -215,6 +214,124 @@ Contains GeoIP information gathered based on the remote_ip field. Only present i
 
 
 
+[float]
+=== `apache2.access.geoip.continent_name`
+
+type: keyword
+
+The name of the continent.
+
+
+[float]
+=== `apache2.access.geoip.country_iso_code`
+
+type: keyword
+
+Country ISO code.
+
+
+[float]
+=== `apache2.access.geoip.location`
+
+type: geo_point
+
+The longitude and latitude.
+
+
+[float]
+=== `apache2.access.geoip.region_name`
+
+type: keyword
+
+The region name.
+
+
+[float]
+=== `apache2.access.geoip.city_name`
+
+type: keyword
+
+The city name.
+
+
+[float]
+== access fields
```

Comment:
I wonder how this slipped to down here. Need to check our generator script.

Reply:
I edited that file myself.